resources/public/api/conf/2.0/application.raml

```diff
@@ -19,6 +19,7 @@ uses:
   types: https://developer.service.hmrc.gov.uk/api-documentation/assets/common/modules/types.raml
 
 /api-platform-test:
+
```

Comment:
?

Reply:
The RAML seems to be valid for online YAML validators.
If you merge the PR, then I will try to publish this API.